python/pyspark/ml/classification.py

```diff
@@ -1630,8 +1614,52 @@ def _to_java(self):
         _java_obj.setPredictionCol(self.getPredictionCol())
         return _java_obj
 
+    def _make_java_param_pair(self, param, value):
```

Comment:
I tried making OneVsRest extend JavaParams but for some reason when I set self._java_obj in the __init__() function as is done with other classes that inherit from JavaParams I get an error from the make_java_param_pair() function in wrapper.py that says that _java_obj does not have an id. From what I can tell the implementation is identical to that of other estimators so I'm not sure what's wrong. To avoid spending too much time on it, I pushed an update with all changes made except for this one.